performance_test/src/experiment_execution/analysis_result.hpp

```diff
@@ -23,14 +23,64 @@
 #include <string>
 
 #include "../utilities/statistics_tracker.hpp"
+#ifdef ODB_FOR_SQL_ENABLED
+  #include <odb/core.hxx>
+  #include "experiment_configuration.hpp"
+#endif
 
 namespace performance_test
 {
 
 /// Outstream operator for timeval to seconds (double).
 std::ostream & operator<<(std::ostream & stream, const timeval & e);
+#ifdef ODB_FOR_SQL_ENABLED
+class RusageTracker
+{
+public:
+  RusageTracker() {}
+
+  RusageTracker(const rusage & sys_usage)
+  : m_ru_utime(sys_usage.ru_utime.tv_sec * 1000000000 + sys_usage.ru_utime.tv_usec * 1000),
```

Comment:
Do this the following way: `std::chrono::seconds(sys_usage.ru_utime.tv_sec)+std::chrono::microseconds(sys_usage.ru_utime.tv_usec))`

Reply:
done